dev-tools/mage/pkg.go

```diff
@@ -121,6 +126,21 @@ func isPackageTypeSelected(pkgType PackageType) bool {
 	return false
 }
 
+// isDockerVariantSelected returns true if SelectedDockerVariants is empty or if
+// docVariant is present on SelectedDockerVariants. It returns false otherwise.
+func isDockerVariantSelected(docVariant DockerVariant) bool {
+	if len(SelectedDockerVariants) == 0 {
+		return true
+	}
+
+	for _, v := range SelectedDockerVariants {
+		if v == docVariant {
+			return true
+		}
+	}
+	return false
```

Comment:
Nit: we can use something more compact:
```suggestion
	return slices.Contains(SelectedDockerVariants, docVariant)
```
https://pkg.go.dev/slices#Contains